index.html

```diff
@@ -20,160 +20,165 @@
   </head>
 
   <body>
-    <div class="header">
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"
+          class="header__logo"
+        />
+      </a>
+
+      <nav>
+        <ul class="navigation__list">
+          <li class="navigation__item">
```

Comment:
on hover of these links, instead of adding an orange background with black colour for the text. You should leave the background as white and then change the link text colour to orange
![Screenshot 2023-09-26 at 09 22 05](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/ec8f4595-5e47-408d-9726-f568b8927e14)